cypress/e2e/gui/validarRodape.cy.js

```diff
@@ -0,0 +1,24 @@
+describe('Validação do Rodapé', () => {
+    it('Validar Rodapé e Links', () => {
+        cy.visit('http://localhost:3000/')
+
+        cy.get('[data-testid="footer"]')
```

Comment:
Que tal algo como o seguinte: `cy.contains('p', 'Copyright 2024 - Talking About Testing').should('be.visible')`?

Reply:
obrigada pela dica